# Schedule

Each day consists of a morning session from 10am to 12pm and an afternoon session from 2pm to 4pm (except for Monday which we start at 11:00). The training week will take place at Imperial, in the room CHEM 660 (except on Thursday which will take place in CHEM 640A).

## Monday
- Morning Session (11:00-12:30)
  - Intorduction to the module, data-driven modelling and statistical learning
- Afternoon Session (2:30-4:30)
  - Discussing group presentations and groups discuss.

## Tuesday
- Morning Session (10:00-12:00)
  - Mechanistic modelling, Bayesian inference and simulation-based inference
- Afternoon Session (2:00-4:00)
  - Python computer lab on simulation-based inference: [ABC Notebook](https://colab.research.google.com/github/CalumGabbutt/CCMI-CWIG-Bayes/blob/main/1_ABC.ipynb)

## Wednesday
- Morning Session (10:00-12:00)
  - Free morning for group presentations and the EDI training
- Afternoon Session (2:00-4:00)
  - Introduction to biological data and single cell RNA-sequencing (scRNA-seq) data analysis
  - R compuer lab on scRNA-seq data science

## Thursday
- Morning Session (10:00-12:00)
  - R compuer lab on scRNA-seq data science (10:00-11:30)
  - Biological background for advanced lab (11:30-12:00)
- Afternoon session (2:00-4:00)
  - Advanced Python and more R computer lab: [SBI Notebook](https://colab.research.google.com/github/CalumGabbutt/CCMI-CWIG-Bayes/blob/main/2_NPE.ipynb)

## Friday
- Morning Session (10:00-12:30)
  - Regression (10:00-10:30)
  - Classification (10:30-11:00)
  - Dimensionality Reduction (11:00-11:30)
  - Clustering (11:30-12:00)
  - Resampling (12:00-12:30)
- Afternoon Session (2:00-4:00)
  - Research talks
    - Freddie Whiting, A mathematical framework to learn evolution of cancer drug resistance from genetic barcoding (2:00-2:30)
    - Amaya Gallagher-Syed,  Benchmarking foundation models for scRNA-seq data (2:30-3:00)
    - Jack Soulsby, Ordered Diffusion Kernels for pseudo-time analysis of scRNA-seq data (3:00-3:30)
    - Dimitris Volteras, Mechanistic models and amortised inference for learning gene-regulation from scRNA-seq data (3:30-4:00)
